src/metabase/email/_header.mustache

```diff
@@ -8,10 +8,10 @@
     }
   </style>
 </head>
-<body class="{{emailType}}" style="font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size: 0.875rem; color: #727479; padding: 1em; background-color: #F9FBFC; ">
+<body class="{{emailType}}" style="font-family: 'Lato', 'Helvetica Neue', Helvetica, sans-serif; padding: 1em;">
```

Comment:
I don't think Lato will actually work here unless the user has it loaded locally. I'd rather not include it so we don't trick ourselves into thinking it's working. We should look into getting it into emails as a separate thing.

Reply:
Yep, good call. Forgot to remove it.